provider/adapters/Cargo.toml

```diff
@@ -29,3 +29,7 @@ serde = { workspace = true, features = ["derive", "alloc"], optional = true }
 icu_provider = { path = "../../provider/core", features = ["macros", "deserialize_json"] }
 icu_locale = { path = "../../components/locale" }
 writeable = { path = "../../utils/writeable" }
+
+[features]
+std = []
+export = ["icu_provider/export", "std"]
```

Comment:
It seems incomplete to not implement `ExportableProvider` on all adapter types.

Reply:
`ForkByMarkerProvider` and `MultiForkByMarkerProvider` are still missing.

Reply:
```
pub type ForkByMarkerProvider<P0, P1> = ForkByErrorProvider<P0, P1, MarkerNotFoundPredicate>;
```